.github/workflows/check.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - name: Setup Gradle
         uses: gradle/gradle-build-action@v2
       - name: Run check with Gradle Wrapper
-        run: ./gradlew :check
```

Comment:
why only store?

Reply:
I wanted to focus just on main module in this diff. Will modify sample next and check all!